package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "root",
+  "private": true,
+  "workspaces": {
+    "packages": [
+      "packages/*"
+    ],
+    "nohoist": [
+      "**/@vue",
```

Comment:
I do not get it,
why are you using `nohoist` for so many libraries?

Reply:
I had issues wih frontend build and functioning.